src/phase.cs

```diff
@@ -0,0 +1,1299 @@
+	[Native]
+	public enum PhasePlaybackMode : long {
+		OneShot = 0,
+		Looping = 1,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseNormalizationMode : long {
+		None = 0,
+		Dynamic = 1,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseCalibrationMode : long {
+		None = 0,
+		RelativeSpl = 1,
+		AbsoluteSpl = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Flags]
+	[Native]
+	public enum PhasePushStreamBufferOptions : ulong {
+		Default = 1uL << 0,
+		Loops = 1uL << 1,
+		Interrupts = 1uL << 2,
+		InterruptsAtLoop = 1uL << 3,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhasePushStreamCompletionCallbackType : long {
+		PushStreamCompletionDataRendered = 0,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseMediumPreset : long {
+		Default = 1835287654,
+		RoomAir = 1835291201,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseMaterialPreset : long {
+		Default = 1833198950,
+		Cardboard = 1833136740,
+		Glass = 1833397363,
+		Brickwork = 1833071211,
+		Concrete = 1833132914,
+		GypsumBoard = 1833400688,
+		Drywall = 1833202295,
+		DoorWoodLight = 1833195340,
+		DoorWoodHeavy = 1833195336,
+		DoorSoundproof = 1833202544,
+		WindowSinglePane = 1834447728,
+		WindowDoublePane = 1834443888,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Flags]
+	[Native]
+	public enum PhaseSpatialPipelineOptions : ulong {
+		DirectPathTransmission = 1uL << 0,
+		EarlyReflections = 1uL << 1,
+		LateReverb = 1uL << 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	public enum PhaseSpatialCategory {
+		[Field ("PHASESpatialCategoryDirectPathTransmission")]
+		DirectPathTransmission,
+		[Field ("PHASESpatialCategoryEarlyReflections")]
+		EarlyReflections,
+		[Field ("PHASESpatialCategoryLateReverb")]
+		LateReverb,
+	}
+
+	[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject), Name="PHASENumericPair")]
+	interface PhaseNumericPair
+	{
+		[Export ("initWithFirstValue:secondValue:")]
+		IntPtr Constructor (double first, double second);
+
+		[Export ("first")]
+		double First { get; set; }
+
+		[Export ("second")]
+		double Second { get; set; }
+	}
+
+	[NoWatch, NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject), Name="PHASEEnvelopeSegment")]
+	interface PhaseEnvelopeSegment
+	{
+/*
+		[Export ("initWithEndPoint:curveType:")]
+		IntPtr Constructor ([unsupported ExtVector: double __attribute__((ext_vector_type(2)))] endPoint, PhaseCurveType curveType);
+
+		[Export ("endPoint", ArgumentSemantic.Assign)]
+		[unsupported ExtVector: double __attribute__((ext_vector_type(2)))] EndPoint { get; set; }
+*/
```

Comment:
Did you want these included?

Reply:
Yes, since it is not supported but we do not want to need to write it.